inc/common.php

```diff
@@ -565,10 +565,15 @@ function ml($id = '', $more = '', $direct = true, $sep = '&amp;', $abs = false)
         $more = str_replace(',', $sep, $more);
     }
 
-    if($abs) {
-        $xlink = DOKU_URL;
-    } else {
-        $xlink = DOKU_BASE;
+    if ($conf[cdn] && $direct) {
```

Comment:
This doesn't throw an error?
Also, the `cdn` option is missing in the configuration file.